src/ResourceManager/Sql/Commands.Sql/help/Stop-AzureRmSqlDatabaseActivity.md

```diff
@@ -0,0 +1,165 @@
+---
+external help file: Microsoft.Azure.Commands.Sql.dll-Help.xml
+Module Name: AzureRM.Sql
+ms.assetid: B5C909D7-6087-463A-83BF-99DD196B9862
+online version: 
+schema: 2.0.0
+---
+
+# Stop-AzureRmSqlDatabaseActivity
+
+## SYNOPSIS
+Cancels the asynchronous operation on the database.
+
+## SYNTAX
+
+```
+Stop-AzureRmSqlDatabaseActivity [-ServerName] <String> [-ElasticPoolName <String>] -DatabaseName <String>
+ [-OperationId <Guid>] [-ResourceGroupName] <String> [-DefaultProfile <IAzureContextContainer>] [-WhatIf]
+ [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Stop-AzureRmSqlDatabaseActivity** cmdlet cancels the asynchronous operation on the database.
+
+## EXAMPLES
+
+### Example 1: Cancel the asynchronous operation on the database
```

Comment:
@jiawang003 for examples, please make sure that you have a sample output added after your cmdlt call so users know what to expect from the returned object